os/mm/kmm_heap/kmm_initialize.c

```diff
@@ -83,7 +83,7 @@ struct mm_heap_s g_kmmheap[CONFIG_KMM_NHEAPS];
 /************************************************************************
  * Public Functions
  ************************************************************************/
-struct mm_heap_s *kmm_get_heap(void)
+struct mm_heap_s *kmm_get_baseheap(void)
```

Comment:
How about moving this into kmm_getheap.c?

Reply:
Moved to `kmm_getheap.c`.
Thanks.